### PR TITLE
set RUNTIME_CONFIG and RUNTIME_SOCKET envars to support new toolkit versions

### PR DESCRIPTION
**Pull request by tariq1890**
This PR includes setting of additional environment variables in container-toolkit asset for forward compatibility with upcoming toolkit releases 

Reference: https://github.com/NVIDIA/nvidia-container-toolkit/pull/704/files

### CODE REVIEW

File: controllers/object_controls.go [hunk 1]
```diff
@@ -1309,6 +1310,9 @@ func transformForRuntime(obj *appsv1.DaemonSet, config *gpuv1.ClusterPolicySpec,
 		sourceSocketFileName := path.Base(runtimeSocketFile)
 		// set envvar for runtime socket
 		var socketEnvvarName string
+		if runtime == gpuv1.Containerd.String() || runtime == gpuv1.Docker.String() {
```
Comment (cdesiniotis) [inline controllers/object_controls.go]: We do not need this conditional. At this point, we know `runtimeSocketFile` is not the empty string, so we can unconditionally set the `RUNTIME_SOCKET` envvar.

Comment (tariq1890) [reply controllers/object_controls.go]: Ah you're right, thanks

File: controllers/object_controls.go [hunk 2]
```diff
@@ -1309,6 +1310,7 @@ func transformForRuntime(obj *appsv1.DaemonSet, config *gpuv1.ClusterPolicySpec,
 		sourceSocketFileName := path.Base(runtimeSocketFile)
 		// set envvar for runtime socket
 		var socketEnvvarName string
+		setContainerEnv(mainContainer, "RUNTIME_SOCKET", DefaultRuntimeSocketTargetDir+sourceSocketFileName)
```
Comment (cdesiniotis) [inline controllers/object_controls.go]: minor nit: can we move this so both `setContainerEnv()` calls are made in consecutive lines? E.g. move this to line 1319

**Review by cdesiniotis — APPROVED** [review]

File: controllers/object_controls.go (hunk 2 above)
Comment (tariq1890) [reply controllers/object_controls.go]: Sure

Comment (tariq1890) [reply controllers/object_controls.go]: Done, please check now

**Pull request merged**